Userland/Libraries/LibWeb/Painting/GradientPainting.h

```diff
@@ -24,7 +24,7 @@ using ColorStopList = Vector<ColorStop, 4>;
 
 struct ColorStopData {
     ColorStopList list;
-    Optional<float> repeat_length;
+    Optional<float> repeat_length; // Either degrees, or CSS Pixels.
```

Comment:
This comment is not correct.
 `repeat_length` is value between 0 and 1 (it's relative to length of the gradient line, like the color stop positions)

Reply:
Oops, that's an old comment from before you changed things for me!